### PR TITLE
Remove duplicate XSAppIconAssets in MacCatalyst template (#1335)

### LINKED ISSUE #1335: Template of Info.plist for MacCatalyst has duplicated XSAppIconAssets entry
### Description

Info.plist has duplicated XSAppIconAssets entry.

### Steps to Reproduce

1.  dotnet new maui-blazor
2.  show Info.plist in MacCatalyst dir

### Expected Behavior

Has one XSAppIconAssets entry

### Actual Behavior

Has two XSAppIconAssets entry

### Basic Information

- Version with issue: Microsoft.Maui.Templates Version: 6.0.100-preview.5.798
- Last known good version: Maybe preview.4
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages: Microsoft.Maui.Templates
- Affected Devices:

### Reproduction Link

Here is the PR to solve this
https://github.com/yamachu/maui/pull/1

PullRequestTemplate shows that

> We are currently only accepting Pull Requests for .NET MAUI issues in our [Handler Property Backlog](https://github.com/dotnet/maui/projects/4). We will continue to update this repository over the next couple of months as we begin to accept more types of PRs.

so I don't send PR yet.

### PR DESCRIPTION
**Pull request by lutzroeder**
### Description of Change ###

Fix #1335

@Eilon @Redth

### CODE REVIEW

**Comment by Redth** [comment]
@mattleibow I think you'd be best to validate this one.

File: src/Templates/src/templates/maui-blazor/MauiApp1/MacCatalyst/Info.plist [hunk 1]
```diff
@@ -28,7 +28,5 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>XSAppIconAssets</key>
-	<string>Assets.xcassets/appicon.appiconset</string>
```
Comment (Eilon) [inline src/Templates/src/templates/maui-blazor/MauiApp1/MacCatalyst/Info.plist:32]: This looks good, also need to make the same change in https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-mobile/MauiApp1/MacCatalyst/Info.plist

**Review by Eilon — CHANGES_REQUESTED** [review]
This definitely looks right, just need to make the same change in the other template.

**Review by Eilon — APPROVED** [review]
Looks good to me.

**Pull request merged**